app/angular/src/client/preview/index.js

```diff
@@ -15,5 +14,9 @@ export const {
   raw,
 } = clientApi;
 
+const framework = 'angular';
```

Comment:
I understand that passing framework will be used later (I see it already in https://github.com/storybookjs/storybook/pull/7119/files#diff-0f7bedc2d2cb7019c3fe0f0f28c18959R13). Are any other usecases to this ? (I am always standing for things to be generic =) )

Reply:
@igor-dv Yeah, I use it to automatically configure things about docs. For example, react defaults to inlining story renders, whereas the rest default to iframes. React and vue support `Props` prop tables, the rest do not, etc.

Reply:
Maybe it worth exposing "feature sets" that each framework supports instead of the framework

Reply:
That's not a bad idea, though I plan to switch on the framework string to figure out e.g. how to extract the prop definitions. In addition, I believe that having this parameter also coincides with the multiframework stuff that @Hypnosphi is working on, so I thought it would just be a good thing to establish while I am making use of it in the docs work, rather than adding some new mechanism.